playwright/login.ts

```diff
@@ -0,0 +1,28 @@
+import { test, TestFixture, Page } from '@playwright/test';
+import * as config from './config.json';
+
+export async function login(page): Promise<void> {
+  await Promise.all([
+    await page.fill('input#account', config.accountLogin),
+    await page.locator('button#login-submit').click(),
+    await page.fill('input#userNameInput', config.email),
+    await page.fill('input#passwordInput', config.password),
+    await page.locator('#submitButton').click(),
+  ]);
```

Comment:
```suggestion
  await page.fill('input#account', config.accountLogin);
  await page.locator('button#login-submit').click();
  await page.fill('input#userNameInput', config.email);
  await page.fill('input#passwordInput', config.password);
  await page.locator('#submitButton').click();
```

Promise.all sert à exécuter plusieurs promesses en parallèle et retourner qu'une fois que l'ensemble des promesses à succeed ou que l'une d'elles à fail

Reply:
Done ✅